src/styles/_catalog.scss

```
@@ -14,6 +14,12 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition: 300ms;
```

```suggestion
    transition-duration: 300ms;
```
Так краще, бо ти більше ніяких опцій не використовуєш

---

specify a `transition-property`